src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__content {
+    padding-inline: 16px;
+  }
+
+  &__title {
+    color: #060b35;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 22px;
+
+    &-text {
+      padding-block: 2px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 14px;
+
+    &-text {
+      padding-block: 1px;
+      color: #616070;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-count {
+      color: #060b35;
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    width: 100%;
+    text-align: center;
+    margin-bottom: 16px;
+
+    &-link {
+      background-color: #00acdc;
+      border: 1px solid #00acdc;
+      border-radius: 5px;
+      display: block;
+      text-decoration: none;
+      text-transform: uppercase;
+      color: #fff;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      padding-block: 12px;
+
+      &:hover {
+        background-color: #fff;
+        color: #00acdc;
+      }
```

Comment:
It's a good practice to change not only the color on hover, but also add some transition to make the change smoother. Consider adding a transition property to the `.card__button-link`.